examples/multigpu_advtrain/trainer.py

```diff
@@ -0,0 +1,211 @@
+import six
```

Comment:
docstring at the top to explain what this file does

Reply:
There is actually still no docstring at the top

Reply:
Fixed in aca841f.